### ISSUE TITLE
broadcast_to op: "invalid shape to broadcast" on 6D tensors

### ISSUE THREAD

**Issue by wookayin** [tensorflow/tensorflow#27850]
**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): Yes
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Linux Ubuntu 16.04
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device: x
- TensorFlow installed from (source or binary): binary
- TensorFlow version (use command below): 1.13.1
- Python version: 3.6
- Bazel version (if compiling from source): -
- GCC/Compiler version (if compiling from source): -
- CUDA/cuDNN version: CUDA 10, cudnn 7.5
- GPU model and memory: Pascal Xp

**Describe the current behavior**

```python
a = tf.zeros([2, 100, 32, 32, 3])
b = tf.broadcast_to( tf.expand_dims(a, 1), [2, 10, 100, 32, 32, 3] )

sess = tf.InteractiveSession()     # or whatever session
sess.run(b).shape
```

gives the error:

```
InvalidArgumentError (see above for traceback): invalid shape to broadcast from [2,1,100,32,32,3] to [2,10,100,32,32,3]
         [[node BroadcastTo_1 (defined at <ipython-input-10-49d2a4dce7c3>:1) ]]
```

**Describe the expected behavior**

Should run normally, giving the `[2, 10, 100, 32, 32, 3]` tensor.

**Other Information**

It works for 5D tensor.

```python
>>> a = tf.zeros([2,100,32,32])
>>> b = tf.broadcast_to( tf.expand_dims(a, 1), [2,10,100,32,32] )

>>> sess.run(b).shape
(2, 10, 100, 32, 32)
```

**Comment by wookayin** [comment]
Using `tf.tile` as a replacement of `tf.broadcast_to` works okay.

```
tf.tile(tf.expand_dims(a, 1), [1, 10, 1, 1, 1, 1])
```

**Comment by Dayananda-V** [comment]
With python 3.6.7 version am able to run the sample snippet and get actual expected o/p.

>>> sess.run(b).shape
(2, 10, 100, 32, 32, 3)

**Comment by wookayin** [comment]
@Dayananda-V Thanks for checking this on yourside. Can you please specify other environment information as well (TF version, it is built from source or from pip/conda, etc.)?

**Comment by saxenasaurabh** [comment]
This should be fixed as of 99121926a8d774f1e205c00108396bc955346e3d
